Sofascore/Sofascore/Views/TabViewItem.swift

```diff
@@ -0,0 +1,71 @@
+import Foundation
+import SwiftUI
+import UIKit
+import SofaAcademic
+import SnapKit
+
+class TabViewItem: BaseView {
+    
+    private let verticalStack: UIStackView = .init()
+    private let sportLogo: UIImageView = .init()
+    private let sportName: UILabel = .init()
```

Comment:
Ovaj sportName je UILabel, pa bi bilo dobro da je sportNameLabel, inace to primjenjujes i to je bas dobra praksa, mozes proci po zadaci pa vidjeti